internal/session/serve.go

```diff
@@ -82,7 +84,29 @@ func (s *Session) serve() {
 
 		case "delete":
 			go s.deleteMachine(ctx, payload)
-
+		case "sethealthy":
```

Comment:
Should we `setHealthy` to be consistent with `updateConfig` below?

Reply:
feel its better to leave separate bc we might sethealthy a lot of components at same time while updateConfig is component specific..